requirements/frozen_docs.py2.txt

```diff
@@ -172,6 +172,7 @@ Werkzeug==0.14.1
 widgetastic.core==0.30.2
 widgetastic.patternfly==0.0.39
 widgetsnbextension==3.4.2
+wrapanapi==3.0.31
```

Comment:
have to update py3 frozen files as well

Reply:
How can I do that? By running freeze_all.py in python3? So I have to download all the requirements for python3?

Reply:
Correct, create a py3 virtualenv ( can use quickstart for it if you want) and activate it, then run freeze_all.

Reply:
Done